rllib/env/multi_agent_env.py

```diff
@@ -401,14 +409,6 @@ def to_base_env(
 
         return env
 
-    @DeveloperAPI
-    def _check_if_space_maps_agent_id_to_sub_space(self) -> bool:
```

Comment:
I think we should only allow the more explicit methods ("_check_if_obs_space_maps_agent_id_to_sub_space" and "_check_if_action_space_maps_agent_id_to_sub_space") to not cause confusion.